civiform/civiform
# Align buttons in edit-or-discard-answers modal

### Description

In the "edit or discard your answers" modal (shown if the user incorrectly fills out a block then clicks "Review" or "Previous"), the buttons had different margins on the right side if the viewport is small enough. That's because the first button had right margin but the second button didn't. This PR moves the margin to be a left margin on the second button.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Created unit and/or browser tests which fail without the change (if possible) -- **existing browser test screenshots updated**


#### User visible changes


### Instructions for manual testing

1. Enable SAVE_ON_ALL_ACTIONS flag.
2. As an applicant, open a block with required questions, don't answer them, and click "Review" or "Previous".
3. Verify that the modal buttons have the same right alignment when the viewport is small enough.

### Issue(s) this completes

Part of #6450
